app/init/launch.ts

```diff
@@ -172,6 +174,15 @@ const launchToHome = async (props: LaunchProps) => {
         }
         case Launch.Normal:
             if (props.coldStart) {
+                const lastViewedChannel = await getLastViewedChannelIdAndServer();
+                const lastViewedThread = await getLastViewedThreadIdAndServer();
+
+                if (lastViewedChannel && lastViewedChannel.server_url === props.serverUrl && lastViewedChannel.channel_id) {
+                    switchToChannelById(props.serverUrl!, lastViewedChannel.channel_id);
+                } else if (lastViewedThread && lastViewedThread.server_url === props.serverUrl && lastViewedThread.thread_id) {
+                    fetchAndSwitchToThread(props.serverUrl!, lastViewedThread.thread_id);
+                }
```

Comment:
according to this, channel takes priority over a thread, but for a user with CRT mode on, it would make sense to have the Thread as priority over the channel imho. can we confirm with PM?

Reply:
agreed for the crt mode, but I think we won't have both as non null values. So, it would be always one or none being truthy, it won't issue might then. your thought?

Reply:
gotcha! I see your point